layers/+emacs/org/config.el

```diff
@@ -95,3 +95,6 @@ ATTENTION: `valign-mode' will be laggy working with tables contain more than 100
 
 (defvar org-enable-roam-server nil
   "if non-nil, enable org-roam-server support")
+
+(defvar org-enable-roam-protocol nil
+  "if non-nil, enable org-roam-protocol (https://www.orgroam.com/manual.html#Roam-Protocol)")
```

Comment:
The documentation should starts with capital letter and ends with period.

The first line of documentation should be a short sentence < 70 characters.